crates/holochain_zome_types/src/element.rs

```diff
@@ -121,8 +121,11 @@ pub enum ElementEntry {
     /// The Header does not contain an entry_address reference.
     NotApplicable,
     /// The Header has an entry but was stored without it.
+    ///
     /// This can happen when you receive gossip of just a header
-    /// when the header type is a [NewEntryHeader]
+    /// when the header type is a [NewEntryHeader] or
+    /// when a [chain query](https://docs.rs/hdk/0.0.104/hdk/chain/fn.query.html) is performed without
```

Comment:
the version can be substituted by _latest_. how about that?

```suggestion
    /// when a [chain query](https://docs.rs/hdk/latest/hdk/chain/fn.query.html) is performed without
```

Reply:
Hm, yeah I see what you mean. I'm thinking to omit it though, and just write hdk::query or something... 

It could be good to put a comment about using 'include_entries' on the `query` comment itself, if there isn't one, and that could be the more useful comment.

Reply:
I chose against `latest` because if the path of the function changes, I want the link to still work. I figured linking to the specific version was fine and that many readers will just click "Go to latest version" in the upper left after following the link